frontend/src/components/Search/useSavedSegments/index.tsx

```diff
@@ -244,179 +326,128 @@ export const useSavedSegments = ({
 						gap="4"
 						userSelect="none"
 					>
-						<IconSolidPlusCircle size={16} color={colors.n9} />
-						Save as a new segment
+						<IconSolidDocumentDuplicate
+							size={16}
+							color={colors.n9}
+						/>
+						Duplicate segment
 					</Box>
 				</Menu.Item>
+				<Menu.Item>
+					<Menu>
+						<Menu.Button
+							style={{
+								paddingLeft: 0,
+							}}
+							size="small"
+							emphasis="low"
+							kind="secondary"
+						>
+							<Box gap="4" display="flex" alignItems="center">
+								<IconSolidSegment size={16} color={colors.n9} />
+								Select other segment
+							</Box>
+						</Menu.Button>
```

Comment:
seems strange that this is a clickable button since it just closes the whole menu and doesn't have an action.

Reply:
Thanks! I added a "preventDefault" on click to avoid closing this menu. I also added it to the "Switch Workspace" button to keep these menus within menus consistent